doc/api/util.md

```diff
@@ -345,6 +345,31 @@ util.format('%% %s');
 Some input values can have a significant performance overhead that can block the
 event loop. Use this function with care and never in a hot code path.
 
+## `util.formatText(format, text)`
+
+> Stability: 1 - Experimental
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `format` {string} A text format defined in `util.inspect.colors`.
+* `text` {string} The text to to be formatted.
+
+This function returns a formatted text considering the `format` passed
+
+```js
+const textRedCollor = util.formatText('red', 'My red message');
+console.log(textRedCollor);
+```
+
+`util.inspect.colors` also provides text formats such as `italic`, and
+`underline`:
+
+```js
+console.log(util.formatText('italic', 'My italic message'));
```

Comment:
nit: I think it makes sense to illustrate with an example does both <u>_italic and underline_</u> since that's what the sentence says, and it's also good to show it can be combined:
````suggestion
`util.inspect.colors` also provides text formats such as `italic`, and
`underline`:

```js
console.log(util.formatText('underline', util.formatText('italic', 'My italic underlined message')));
````